app/services/google/sheets.py

```
@@ -0,0 +1,243 @@
+    def _fetch():
+        session = _get_sheets_session()
+        if not session:
+            return []
+        try:
+            result = _get_json(
+                session,
+                f"https://sheets.googleapis.com/v4/spreadsheets/{spreadsheet_id}",
+                params={"fields": "sheets.properties.title"},
+            )
+            return [sheet["properties"]["title"] for sheet in result.get("sheets", [])]
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)